test/adapt/proto.ts

```diff
@@ -0,0 +1,352 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as assert from 'assert';
+import {describe, it} from 'mocha';
+import {protobuf} from 'google-gax';
+import * as adapt from '../../src/adapt';
+import {TableSchema} from '@google-cloud/bigquery';
+
+const {Root} = protobuf;
+
+describe('Adapt Protos', () => {
```

Comment:
If you're interested, I recently discovered a package `snap-shot-it` that lets you just record the output of a test, and then it will compare against the saved one. Not super essential though.